tests/test_thunderbird.py

```diff
@@ -144,38 +175,60 @@ def add_local_account(tb):
 
     # set outgoing server
     settings.childNamed('Outgoing Server (SMTP)').doActionNamed('activate')
-    settings.button('Add.*').doActionNamed('press')
-    add_server = tb.dialog('SMTP Server')
+    smtp_settings = settings.findChild(
+        GenericPredicate(name='Outgoing Server (SMTP) Settings',
+                         roleName='document web'))
```

Comment:
This should be under `if version >= 78`. possibly as an alternative to the call above. Now it fails the test on TB 68.